### PR TITLE
Merge develop-2.2 into riak_ts-develop

### PR DESCRIPTION
**Pull request by korry8911**
So I made a branch off of riak_ts-develop, merged in develop-2.2 which required resolving 3 conflicts. @javajolt

### CODE REVIEW

File: rebar.config [hunk 1]
```diff
@@ -11,6 +11,7 @@
 {eunit_opts, [verbose]}.
 
 {deps, [
+<<<<<<< HEAD
```
Comment (hazen) [inline rebar.config]: @korry8911 We have merge-conflict here.  You'll want the deps in the `HEAD` section except for the addition of the `rebar_lock_deps_plugin` we just added.  That needs to come from the `develop-2.2` side.

File: src/riak_test_escript.erl [hunk 2]
```diff
@@ -312,27 +312,46 @@ run_test(Test, TestType, Outdir, TestMetaData, Report, HarnessArgs, NumTests) ->
...
         _ ->
-            {value, {log, L}, TestResult} = lists:keytake(log, 1, SingleTestResult),
+            {value, {log, L}, TestResult} =
+                lists:keytake(log, 1, SingleTestResult),
             case giddyup:post_result(TestResult) of
                 error -> woops;
                 {ok, Base} ->
                     %% Now push up the artifacts, starting with the test log
                     giddyup:post_artifact(Base, {"riak_test.log", L}),
+<<<<<<< HEAD
```
Comment (hazen) [inline src/riak_test_escript.erl]: Another conflict.  Here we want to take the `develop-2.2` side instead.

File: src/riak_test_escript.erl [hunk 3]
```diff
@@ -312,27 +312,46 @@ run_test(Test, TestType, Outdir, TestMetaData, Report, HarnessArgs, NumTests) ->
...
+                                        [{giddyup_url, list_to_binary(Base)}],
+                    [rt:post_result(ResultPlusGiddyUp, WebHook) ||
+                     WebHook <- get_webhooks()]
+>>>>>>> develop-2.2
             end
     end,
     rt_cover:stop(),
     [{coverdata, CoverageFile} | SingleTestResult].
 
+<<<<<<< HEAD
```
Comment (hazen) [inline src/riak_test_escript.erl]: Conflict. Here we actually want to keep both sides.

**Review by hazen — CHANGES_REQUESTED** [review]
Once the conflicts are resolved, we should be good to go

**Comment by korry8911** [comment]
ok, I think the conflicts have been resolved correctly now. @javajolt

File: src/riak_test_escript.erl [hunk 4]
```diff
@@ -325,14 +325,12 @@ run_test(Test, TestType, Outdir, TestMetaData, Report, HarnessArgs, NumTests) ->
                 {ok, Base} ->
                     %% Now push up the artifacts, starting with the test log
                     giddyup:post_artifact(Base, {"riak_test.log", L}),
-<<<<<<< HEAD
                     [ giddyup:post_artifact(Base, File) || File <- rt:get_node_logs() ],
```
Comment (hazen) [inline src/riak_test_escript.erl:328]: Lines 328-333 need to be removed.  This was an either/or decision and we are choosing _only_ the `develop-2.2` changes

File: rebar.config [hunk 5]
```diff
@@ -20,18 +19,9 @@
...
-        {meck, "0.8.2", {git, "git://github.com/basho/meck.git", {tag, "0.8.2"}}},
-        {mapred_verify, ".*", {git, "git://github.com/basho/mapred_verify", {branch, "master"}}},
-        {riakc, ".*", {git, "git://github.com/basho/riak-erlang-client", {branch, "develop-2.2"}}},
-        {riakhttpc, ".*", {git, "git://github.com/basho/riak-erlang-http-client", {branch, "develop-2.2"}}},
-        {kvc, "1.3.0", {git, "https://github.com/etrepum/kvc", {tag, "v1.3.0"}}},
-        {druuid, ".*", {git, "git://github.com/kellymclaughlin/druuid.git", {tag, "0.2"}}},
+        {tdiff,         ".*",   {git, "https://github.com/basho/tdiff",                   {tag, "0.1"}}},
         {rebar_lock_deps_plugin, ".*", {git, "https://github.com/basho/rebar_lock_deps_plugin.git", {tag, "3.1.0p1"}}}
->>>>>>> develop-2.2
+
```
Comment (hazen) [inline rebar.config]: We can kill this blank line

**Review by hazen — CHANGES_REQUESTED** [review]
Once these change have been made:
- build your branch
- run a simple test, like basic-command-line to see that it works

Remember this is against: Riak TS

**Comment by korry8911** [comment]
ok, build riak_test and ran basic_command_line:
Test Results:

## basic_command_line-riak_kv_bitcask_backend,riak_kv_eleveldb_backend: pass

0 Tests Failed
1 Tests Passed
That's 100.0% for those keeping score
@javajolt

**Review by hazen — APPROVED** [review]

**Comment by hazen** [comment]
:+1:

**Pull request merged**